ElementX/Sources/Services/Authentication/AuthenticationServiceProxy.swift

```diff
@@ -122,6 +122,18 @@ class AuthenticationServiceProxy: AuthenticationServiceProxyProtocol {
                                                      deviceId: deviceID)
             }
             
+            let refreshToken = try? await Task.dispatch(on: .global()) {
+                try client.session().refreshToken
+            }
+            
+            if refreshToken != nil {
+                MXLog.warning("Refresh token found for a non oidc session, can't restore session, logging out")
+                _ = try? await Task.dispatch(on: .global()) {
+                    try client.logout()
+                }
```

Comment:
Not at all important, but may as well simplify it to this.
```suggestion
                _ = await Task.dispatch(on: .global()) {
                    try? client.logout()
                }
```